.teamcity/Gradle_Check/projects/RootProject.kt

```diff
@@ -33,6 +33,8 @@ class RootProject(model: CIBuildModel, functionalTestBucketProvider: FunctionalT
 
     params {
         param("env.GRADLE_ENTERPRISE_ACCESS_KEY", "%ge.gradle.org.access.key%")
+        param("env.NPM_CONFIG_REGISTRY", "https://repo.gradle.org/gradle/api/npm/npm-temporary/")
```

Comment:
If the build works, I think we should add this environment variables to the Gradle project manually via the UI, since we run Gradle builds outside of Check for promotion and ad-hoc performance tests as well.